### ISSUE TITLE
No info on the enchantments and how to use them

### ISSUE THREAD

**Issue by SamayaNL** [AmyMialeeMods/glide#4]
I cannot seem to find any info on how to use the enchantments. It would be great if they had tooltips with info and an explanation.

other than that, I love this little mod, one of the best.

**Comment by AmyMialee** [comment]
To use them all you just use the glider twice.

**Comment by SamayaNL** [comment]
> To use them all you just use the glider twice.

I am trying but it doesn't seem to be working. Do the enchantments need to be on the glider itself? Because it let me enchant any piece of armour. So I didn't put it on the glider.

**Comment by AmyMialee** [comment]
yeah it's supposed to be on the glider

**Comment by SamayaNL** [comment]
Ah ok makes sense. Is it a bug that it lets you use the enchants on
whatever?

**Comment by AmyMialee** [comment]
yeah it'll be fixed if I can get the motivation to update my mods

**Comment by SamayaNL** [comment]
Understandable. Take care
